TrackingTools/TrajectoryState/BuildFile.xml

```diff
@@ -6,11 +6,11 @@
 <use   name="DataFormats/TrackReco"/>
 <use   name="DataFormats/TrajectoryState"/>
 <use   name="DataFormats/BeamSpot"/>
+<use   name="MagneticField/Engine"/>
 <use   name="TrackingTools/AnalyticalJacobians"/>
 <use   name="TrackingTools/TrajectoryParametrization"/>
-<use   name="MagneticField/Engine"/>
 
-<use   name="vdt_headers"/>
+<use   name="vdt"/>
```

Comment:
Do not make this change.  The 7_4_X dependency is the correct one, and I already have a PR that changes the 7_4_ROOT6_X version of this file to the 7_4_X version